docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -167,18 +145,63 @@ The configuration parameters which must be set are:
   - This is set to the URI of the shared storage system to mount to the Alluxio root. This shared
   shared storage system must be accessible by all master nodes and all worker nodes.
   - Examples: `alluxio.underfs.address=hdfs://1.2.3.4:9000/alluxio/root/`, `alluxio.underfs.address=s3a://bucket/dir/`
+  
+### Setting up HA cluster with internal leader election
+
+The minimal configuration to set up a HA cluster is to give the master rpc addresses to all nodes inside the cluster.
```

Comment:
Looks like we are recommending setting `alluxio.master.embedded.journal.addresses` in the server side and recommend setting `alluxio.master.rpc.addresses` on the client side.

Why can't we directly use the `alluxio.master.rpc.addresses` on the server and client side, and let `alluxio.master.embedded.journal.addresses` derived from the `alluxio.master.rpc.addresses` when we have multiple master and zookeeper is not enabled.

In the HA mode client URI, the values we set are also `alluxio.master.rpc.addresses`.

Reply:
`alluxio.master.rpc.addresses` is the client side property for connecting to the master RPC service. `alluxio.master.embedded.journal.addresses` is the server side property for telling masters how to find each other for running the embedded journal service. I think it makes more sense for clients to find servers based on server configuration than for servers to find themselves based on client configuration.

We set `alluxio.master.rpc.addresses` in HA mode because we're on the client side, and that's the client side property.

One use case where `rpc` could be different from `embedded` is in changing masters without requiring clients to restart. To change from `embedded=[addr1, addr2, addr3` to `embedded=[addr3, addr4, addr5]`, we could first set `rpc=[addr1, addr2, addr3, addr4, addr5]`. Clients configured that way could find both the original set of masters and the changed set of masters. In this case, it wouldn't make sense or work to have `embedded` derived from `rpc`, but it would work for `rpc` to be derived from `embedded`

Reply:
Got it!
Now I suggest setting the `alluxio.master.embedded.journal.addresses` in the server side and set the `alluxio.master.rpc.addresses` in the client side. Thanks